### PR TITLE
Prepare release v0.7.0

### PR DESCRIPTION
**Pull request by StanczakDominik**
Waiting for tests to pass...

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -9,12 +9,11 @@ license = BSD+Patent
...
-    Changes = https://docs.plasmapy.org/en/stable/whatsnew/index.html
-    Source Code = https://github.com/plasmapy/plasmapy
-    Issue Tracker = https://github.com/plasmapy/plasmapy/issues/
-    Twitter = https://twitter.com/PlasmaPy
-    Chat = "https://app.element.io/#/room/#plasmapy:openastronomy.org"
+    Documentation=https://docs.plasmapy.org/
+    Changelog=https://docs.plasmapy.org/en/stable/whatsnew/index.html
+    Source=https://github.com/plasmapy/plasmapy
+    Issues=https://github.com/plasmapy/plasmapy/issues/
+    Twitter=https://twitter.com/PlasmaPy
```
Comment (StanczakDominik) [inline setup.cfg:16]: I had to modify these, notably removing the Chat one, as Twine was giving me problems on upload - as documented at https://github.com/pypa/warehouse/issues/10198.

**Comment by StanczakDominik** [comment]
Should be good to go! :)

File: .mailmap [hunk 2]
```diff
@@ -51,21 +59,31 @@ Samaiyah I. Farid <38890767+samaiyahfarid@users.noreply.github.com>
...
+dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>
 flaixman <biel_nadal@hotmail.com>
 goodab <goodab@uw.edu>
+haman80 <70356097+haman80@users.noreply.github.com>
 hzxusx <hzxusx@gmail.com>
 jams2 <me@joshuamunn.com>
 lgoenner <lgoenner@users.noreply.github.com>
 misupova <marija.isupova@gmail.com>
 nrb1324 <nrb1324@hotmail.com>
+pre-commit-ci[bot] <66853113+pre-commit-ci[bot]@users.noreply.github.com>
```
Comment (namurphy) [inline .mailmap:84]:
```suggestion
flaixman <biel_nadal@hotmail.com>
goodab <goodab@uw.edu>
haman80 <70356097+haman80@users.noreply.github.com>
hzxusx <hzxusx@gmail.com>
jams2 <me@joshuamunn.com>
lgoenner <lgoenner@users.noreply.github.com>
misupova <marija.isupova@gmail.com>
nrb1324 <nrb1324@hotmail.com>
```
[optional] Not a big deal, but we could remove dependabot and pre-commit-ci from the mailmap...though I suppose they both have a right to be included in author lists, perhaps?  🤔

**Review by namurphy — APPROVED** [review]
Looks great to me!  One minor optional suggestion, but I'd say it's ready to be merged anytime.

**Pull request merged**